pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -63,30 +63,31 @@ const (
-	SchedulerName           string                               `json:"schedulerName,omitempty"`
-	ClusterServiceDNSSuffix string                               `json:"clusterServiceDNSSuffix,omitempty"`
-	ClusterServiceDNSMode   DnsMode                              `json:"clusterServiceDNSMode,omitempty"`
-	Sharding                Sharding                             `json:"sharding,omitempty"`
-	InitImage               string                               `json:"initImage,omitempty"`
-	MultiCluster            MultiCluster                         `json:"multiCluster,omitempty"`
-	TLS                     *TLSSpec                             `json:"tls,omitempty"`
+	Pause                        bool                                 `json:"pause,omitempty"`
+	Unmanaged                    bool                                 `json:"unmanaged,omitempty"`
+	CRVersion                    string                               `json:"crVersion,omitempty"`
+	Platform                     *version.Platform                    `json:"platform,omitempty"`
+	Image                        string                               `json:"image"`
+	ImagePullSecrets             []corev1.LocalObjectReference        `json:"imagePullSecrets,omitempty"`
+	UnsafeConf                   bool                                 `json:"allowUnsafeConfigurations,omitempty"`
+	IgnoreLabels                 []string                             `json:"ignoreLabels,omitempty"`
+	IgnoreAnnotations            []string                             `json:"ignoreAnnotations,omitempty"`
+	Mongod                       *MongodSpec                          `json:"mongod,omitempty"`
+	Replsets                     []*ReplsetSpec                       `json:"replsets,omitempty"`
+	Secrets                      *SecretsSpec                         `json:"secrets,omitempty"`
+	Backup                       BackupSpec                           `json:"backup,omitempty"`
+	ImagePullPolicy              corev1.PullPolicy                    `json:"imagePullPolicy,omitempty"`
+	PMM                          PMMSpec                              `json:"pmm,omitempty"`
+	UpdateStrategy               appsv1.StatefulSetUpdateStrategyType `json:"updateStrategy,omitempty"`
+	UpgradeOptions               UpgradeOptions                       `json:"upgradeOptions,omitempty"`
+	SchedulerName                string                               `json:"schedulerName,omitempty"`
+	ClusterServiceDNSSuffix      string                               `json:"clusterServiceDNSSuffix,omitempty"`
+	ClusterServiceDNSMode        DnsMode                              `json:"clusterServiceDNSMode,omitempty"`
+	Sharding                     Sharding                             `json:"sharding,omitempty"`
+	InitImage                    string                               `json:"initImage,omitempty"`
+	InitContainerSecurityContext *corev1.SecurityContext              `json:"initContainerSecurityContext,omitempty"`
```

Comment:
Please add this option to cr.yaml

Reply:
check the test security-context and add test case

Reply:
> Please add this option to cr.yaml

@hors besides this we should add `initImage` field to cr.yaml as well, what should we put as an actual image?

Reply:
@inelpandzic please see example https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L23

Reply:
Aha, cool, thx @hors